lighthouse-core/gather/drivers/driver.js

```diff
@@ -123,6 +123,14 @@ class Driver {
     return this._connection.sendCommand(method, params);
   }
 
+  /**
+   * Get current tab id
+   * @return {!Promise<string>}
+   */
+  getCurrentTabId() {
+    return Promise.reject(new Error('Not implemented'));
```

Comment:
looks like the issue is here. `Driver` no longer extends `ExtensionConnection` or `CriConnection`, so I believe driver's `getCurrentTabId` needs to call `this._connection.getCurrentTabId()`